lib/src/components/NxBinaryDonutChart/__tests__/NxDonutChart.test.tsx

```diff
@@ -5,15 +5,46 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import NxBinaryDonutChart from '../NxBinaryDonutChart';
+import NxBinaryDonutChart, {Props} from '../NxBinaryDonutChart';
 
 describe('NxBinaryDonutChart', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxBinaryDonutChart, {percent: 0});
+  const minimalProps: Props = {
+    value: 0,
+    percent: 0
+  };
```

Comment:
Only one or the other of these props is supposed to be specified at a time.  I recommend having two separate sets of props: `minimalProps` and `minimalDeprecatedProps`, with a `getShallowComponent`-like function constructed from each.

Reply:
This is updated here a566b81d0357d211dcaf4fc5849f33401a6d0b0f with two sets of props and corresponding shallowComponents.